src/data/local/github/tracks-node/track-5/steps/step-4.md

```diff
@@ -0,0 +1,16 @@
+# Step 4/6 - Database and environments
+### Estimated time: 1 hour
+
+Now we're going to include the database on our tests. After all, we should test if our queries will be returning what is on the database correctly, and if our mutations are changing the database accordingly. 
+
+Do you remember we talked about setting up 2 containers because we were going to use 2 databases? Yeah, the north remembers 🐺! As a preparation for this step, you're going to prepare 2 environments: the one you're already using to develop with `npm start` and one additional to run the tests. Do some reasearch about the importance of having separate environments on our development workflow. We will talk about it sometime on our meetings too.
```

Comment:
```suggestion
Do you remember we talked about setting up 2 containers because we were going to use 2 databases? Yeah, the north remembers 🐺! As a preparation for this step, you're going to prepare 2 environments: the one you're already using to develop with `npm start` and one additional to run the tests. Do some research about the importance of having separate environments on our development workflow. We will talk about it on our meetings too.
```